pkg/cloud/azure/services/availabilitysets/availabilitysets.go

```diff
@@ -21,6 +24,42 @@ func (s *Service) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
 	if !ok {
 		return errors.New("invalid availability set specification")
 	}
+	// Get the MaximumPlatformFaultDomainCount
+	skuService := resourceskus.NewService(s.Scope)
+	skuSpec := resourceskus.Spec{
+		Name:         string(compute.AvailabilitySetSkuTypesAligned),
+		ResourceType: resourceskus.AvailabilitySets,
+	}
+
+	skuI, err := skuService.Get(ctx, skuSpec)
+	if err != nil {
+		if errors.Is(err, resourceskus.ErrResourceNotFound) {
+			return machinecontroller.InvalidMachineConfiguration("failed to obtain availability set information for Aligned '%s' from Azure: %s", skuSpec.Name, err)
```

Comment:
what does "Aligned" mean here?

Reply:
@elmiko Thank you for helping review, I updated, "Aligned" mean Aligned SKU type.